pkg/controller/nodenetworkconfigurationpolicy/enactmentstatus/conditions/counter.go

```diff
@@ -10,19 +10,25 @@ import (
 
 type CountByConditionType map[nmstatev1alpha1.ConditionType]int
 
-func Count(enactments nmstatev1alpha1.NodeNetworkConfigurationEnactmentList) CountByConditionType {
-	trueConditionsCount := CountByConditionType{}
+type ConditionCount map[corev1.ConditionStatus]CountByConditionType
+
+func Count(enactments nmstatev1alpha1.NodeNetworkConfigurationEnactmentList) ConditionCount {
+	conditionCount := ConditionCount{
```

Comment:
Could you please switch it so it is conditionsCount.Available.True instead of conditionsCount.True.Available. The structure seems odd.